common/source/docs/common-autopilots.rst

```diff
@@ -16,6 +16,7 @@ Open hardware
     The Cube <common-thecube-overview>
     Pixracer <common-pixracer-overview>
     CUAV v5 <common-pixhackV5-overview>
+    CUAV v5 Nano <common-cuav-v5nano-overview>
```

Comment:
missing a link to v5+ plage?

Reply:
Yes. I don’t know whether to deprecate v5 and just have v5-plus, or have both.
Probably no harm having both until I find out.